docs/understanding-docker.md

```diff
@@ -17,274 +17,287 @@ weight = -90
-and testing
-* Deploy those applications to your production environment,
- whether it is in a local data center or the Cloud
+  and testing
+* Deploy those applications to your production environment, whether it is in a
+  local data center or the Cloud
 
 ## What is Docker Engine?
 
-Docker Engine is a client-server application with these major components:
+_Docker Engine_ is a client-server application with these major components:
 
 * A server which is a type of long-running program called a daemon process.
 
-* A REST API which specifies interfaces that programs can use to talk to the daemon and instruct it what to do.
+* A REST API which specifies interfaces that programs can use to talk to the
+    daemon and instruct it what to do.
 
 * A command line interface (CLI) client.
 
 ![Docker Engine Components Flow](article-img/engine-components-flow.png)
 
-The CLI makes use of the Docker REST API to control or interact with the Docker daemon through scripting or direct CLI commands. Many other Docker applications make use of the underlying API and CLI.
+The CLI uses the Docker REST API to control or interact with the Docker daemon
+through scripting or direct CLI commands. Many other Docker applications use the
+underlying API and CLI.
 
-The daemon creates and manages Docker objects.  Docker objects include images, containers, networks, data volumes, and so forth.
+The daemon creates and manages Docker _objects_, such as images, containers,
+networks, and data volumes.
 
 > **Note:** Docker is licensed under the open source Apache 2.0 license.
 
 ## What can I use Docker for?
 
-*Faster delivery of your applications*
+*Fast, consistent delivery of your applications*
 
-Docker is perfect for helping you with the development lifecycle. Docker
-allows your developers to develop on local containers that contain your
-applications and services. It can then integrate into a continuous integration and
-deployment workflow.
+Docker can streamline the development lifecycle by allowing developers to work in
+standardized environments using local containers which provide your applications
+and services. You can also integrate Docker into your continuous integration and
+continuous deployment (CI/CD) workflow.
 
-For example, your developers write code locally and share their development stack via
-Docker with their colleagues. When they are ready, they push their code and the
-stack they are developing onto a test environment and execute any required
-tests. From the testing environment, you can then push the Docker images into
-production and deploy your code.
+Consider the following example scenario. Your developers write code locally and
+share their work with their colleagues using Docker containers. They can use
+Docker to push their applications into a test environment and execute automated
+and manual tests. When problems are found, they can be fixed and the updated code
```

Comment:
nit: active could improve this: When developers find problems, they can fix them and update the code....